generators/cypress/templates/src/test/javascript/cypress/integration/account/login_page_spec.ts.ejs

```diff
@@ -0,0 +1,41 @@
+import { titleLoginSelector, errorLoginSelector, usernameLoginSelector, passwordLoginSelector, submitLoginSelector } from '../../support/commands';
+
+describe('login modal', () => {
+  before(() => {
+    cy.visit('');
+    cy.clickOnLoginItem();
+  });
+
+  after(() => {
```

Comment:
You should try to avoid as much as possible after hooks: https://docs.cypress.io/guides/references/best-practices.html#Using-after-or-afterEach-hooks